Binary files orig/ISoLDE/src/.Isolde.c.swp and patched/ISoLDE/src/.Isolde.c.swp differ
diff -Nru orig/ISoLDE/src/Isolde.c patched/ISoLDE/src/Isolde.c
--- orig/ISoLDE/src/Isolde.c	2023-10-20 20:30:18.000000000 +0200
+++ patched/ISoLDE/src/Isolde.c	2024-03-16 11:36:24.456478100 +0100
@@ -24,12 +24,12 @@
 	double *crit9glob = REAL(R_crit9glob), *diff9glob = REAL(R_diff9glob), *denom9glob = REAL(R_denom9glob), *distdiffpropH0 = REAL(R_distdiffpropH0);
 	int *nNRNF = INTEGER(R_NRowReadNormFil), *Nboot = INTEGER(R_Nboot), *NCore = INTEGER(R_NCore), *WithRepeat = INTEGER(R_WithRepeat); // *ChoixRep1 = INTEGER(R_TChoixRep1);
 
-	int sNRNF = length(R_NbReadNormFil); int cNRNF = floor(sNRNF / nNRNF[0]);
+	int sNRNF = Rf_length(R_NbReadNormFil); int cNRNF = floor(sNRNF / nNRNF[0]);
 	int i, j, k, line, col, NbreDDPH0, Nbre2DDPH0, NColDDPH0, NumClasse, NbreVecProb, numBoot; //m, 
 	double *vecBRNF, *vecDDPH0, TempDDPH0, *vecEC9B; //, *vecEC9BZ
 	double AleaSelect, nu0;   //  tempc, tempd, tempe, 
 	
-	int nIndM1 = length(VECTOR_ELT(R_IndM, 0)); int nIndM2 = length(VECTOR_ELT(R_IndM, 1)); int nIndM = nIndM1 + nIndM2; 
+	int nIndM1 = Rf_length(VECTOR_ELT(R_IndM, 0)); int nIndM2 = Rf_length(VECTOR_ELT(R_IndM, 1)); int nIndM = nIndM1 + nIndM2; 
 	int nIndP1 = nIndM1; int nIndP2 = nIndM2; int nIndP = nIndP1 + nIndP2; 
 	int *IndM, *IndM1, *IndM2, *IndP, *IndP1, *IndP2, *ChoixRepM, *ChoixRepP, *NbBootOk, *IndMPv2;	
 
@@ -376,12 +376,12 @@
 	//double *ChoixRep3 = REAL(R_TChoixRep3);
 	int *nNRNF = INTEGER(R_NRowReadNormFil), *nNRH1 = INTEGER(R_NRowNbreadH1), *Nboot2 = INTEGER(R_Nboot2), *NCore = INTEGER(R_NCore);
 
-	int sNRNF = length(R_NbReadNormFil); int cNRNF = floor(sNRNF / nNRNF[0]);
+	int sNRNF = Rf_length(R_NbReadNormFil); int cNRNF = floor(sNRNF / nNRNF[0]);
 	int i, j, line, col, numBoot; 
 	double *vecBRNF, *vecNRH1, *vecEDH1, *vecEC9BH1B, *vecNRTT; //, *vecEC9BH1BZ
 	double lambda0; //   tempc, tempd, tempe, intpart, AleaSelect
 	
-	int nIndM1 = length(VECTOR_ELT(R_IndM, 0)); int nIndM2 = length(VECTOR_ELT(R_IndM, 1)); int nIndM = nIndM1 + nIndM2; 
+	int nIndM1 = Rf_length(VECTOR_ELT(R_IndM, 0)); int nIndM2 = Rf_length(VECTOR_ELT(R_IndM, 1)); int nIndM = nIndM1 + nIndM2; 
 	int nIndP1 = nIndM1; int nIndP2 = nIndM2; int nIndP = nIndP1 + nIndP2; 
 	int *IndM1v2, *IndM2v2, *IndP1v2, *IndP2v2;
 	int *IndM, *IndM1, *IndM2, *IndP, *IndP1, *IndP2, *ChoixRepM, *ChoixRepP, *NbBootOk, *IndMPv2;	
diff -Nru orig/ISoLDE/src/Makevars patched/ISoLDE/src/Makevars
--- orig/ISoLDE/src/Makevars	2023-10-20 20:30:18.000000000 +0200
+++ patched/ISoLDE/src/Makevars	2024-03-16 11:34:04.137174500 +0100
@@ -1,2 +1,3 @@
 PKG_CFLAGS = $(SHLIB_OPENMP_CFLAGS)
 PKG_LIBS = $(SHLIB_OPENMP_CFLAGS)
+PKG_CPPFLAGS = -DR_NO_REMAP
